samples/subsys/power/stm32_pm/sample.yaml

```diff
@@ -0,0 +1,17 @@
+sample:
+  name: Low Power Timer Sample for nucleo board
+tests:
+  sample.power.pm.stm32_lptim.lsi:
```

Comment:
This might be better placed at `samples/boards/stm32/lptim` or otherwise under `samples/boards/stm32`, as it's demonstrating something related to how a particular family of boards works, and several other systems (TI, Nordic, mec15xxevb) have placed power management demonstrations there.

Reply:
moving to a new `samples/boards/stm32/pm_lptim`